### ISSUE TITLE
`requirements-lint.txt` includes `torch` which adds 5GB to the venv folder

### ISSUE THREAD

**Issue by jleibs** [rerun-io/rerun#5542]
This means our docker container that we use for linting requiring pulling an extra 5Gigs just so we can validate our torch type signatures.

**Comment by Wumpf** [comment]
it's unfortunate but we do want to check torch type signatures. What do you suggest? Somehow only including torch type information without the torch library itself?

**Comment by jleibs** [comment]
Yeah, not sure, but it's a comical amount of gratuitous network overhead for something relatively small.

I've used some tricks in docker layers before to figure out the files that never get accessed and then purge them from the layer. It would cause lots of cuda imports that we don't care about to fail but would probably do the trick for our basic tests.
